tests/unit/helpers.js

```diff
@@ -241,6 +241,12 @@ export function createFakeEvent(extraProps = {}) {
   };
 }
 
+export function createFakeChangeEvent(value = '') {
```

Comment:
Typing `createFakeEvent({ target: { value: '' } })` seems fine to me. Why create a new helper?